src/file-upload/internal.tsx

```diff
@@ -95,7 +95,7 @@ function InternalFileUpload(
     setNextFocusIndex(removeFileIndex);
   };
 
-  const isDropzoneVisible = useDropzoneVisible(multiple);
+  const { isFileDragging } = useFileDragging();
```

Comment:
I would suggest we rename the utility to `useFilesDragging` as technically there can be more than one file dragging and the plural form accounts for it better imo. Likewise, I recommend that we rename the response attribute to `isDragging` or `isFilesDragging`.

Reply:
I'll do `areFilesDragging` for grammatical purposes 😄 otherwise, sounds good.